src/ert/gui/tools/plot/plottery/plots/ensemble.py

```diff
@@ -86,9 +86,16 @@ def _plotLines(
         if len(data) == 1 and not style.marker:
             style.marker = "."
 
+        if plot_config.inverted_axes:
+            x = data.to_numpy()
+            y = data.index.to_numpy()
+            axes.yaxis.set_inverted(True)
```

Comment:
yes, `inverted` here means reverse the direction, so we should not mix usage of the term.